File: problems/algorithms/3223/3223_minlength.c
int minimumLength(char* s) {
    int arr[26] = {0};

    while(*s != '\0'){
        arr[*s - 'a'] ++;
        s++;
    }

    int res =0;
    for(int i=0; i<26; i++){
        if (arr[i] == 0){
            continue;
        }
        else if (arr[i] % 2 == 0){
            res += 2;
        }
        else{
            res += 1;
        }        
    }
    return res;
}

File: problems/algorithms/3223/3223_minlength.py
class Solution:
    def minimumLength(self, s: str) -> int:
        pos = [0] * 26
        for letter in s:
            pos[ord(letter) - ord('a')] += 1 

        cpt = 0
        for n in pos:
            if n == 0:
                continue
            elif n % 2 == 0:
                cpt += 2
            else:
                cpt += 1

        return cpt